docs/ipsec-tunnel.md

```diff
@@ -1,16 +1,17 @@
 # IPsec Encryption of Tunnel Traffic with Antrea
 
-Antrea supports encrypting tunnel traffic across Nodes with IPsec ESP. At this
-moment, IPsec encyption works only for GRE tunnel (but not Geneve, VXLAN, and
-STT tunnel types).
+Antrea supports encrypting tunnel traffic across Nodes with IPsec ESP.
+IPsec encyption works for all OVS supported tunnel including Geneve, GRE,
+VXLAN, and STT tunnel.
 
 ## Prerequisites
 
 IPsec requires a set of Linux kernel modules. Check the required kernel modules
 listed in the [strongSwan documentation](https://wiki.strongswan.org/projects/strongswan/wiki/KernelModules).
 Make sure the required kernel modules are loaded on the Kubernetes Nodes before
 deploying Antrea with IPsec encyption enabled.
-
+If you want to enable IPsec with Geneve, please make sure [this commit](https://github.com/torvalds/linux/commit/34beb21594519ce64a55a498c2fe7d567bc1ca20) is included
```

Comment:
So do VXLAN and STT work with older 18.04 versions, without the commit?

Reply:
Add a new line before this paragraph.

Reply:
yes, VXLAN works, STT module is not in the Linux tree, so need extra step to install it manually from source, I have updated [here](https://github.com/antrea-io/antrea/blob/main/docs/getting-started.md#ensuring-requirements-are-satisfied) last time, let me add it here as well.

Reply:
done

Reply:
I do not see you added a new line.

Reply:
ah, misunderstood the new line, I added it before `## Installation`, will add it.

Reply:
done